Objects/obmalloc.c

```diff
@@ -967,6 +968,12 @@ static int running_on_valgrind = -1;
 
 typedef struct _obmalloc_state OMState;
 
+/* obmalloc state for main interpreter and shared by all interpreters without
+ * their own obmalloc state.  By not explicitly initalizing this structure, it
+ * will be allocated in the BSS which is a small performance win.  The radix
```

Comment:
How much of a difference is it?  IIRC, we used a static initializer in 3.11.

I ask because things get a little bit simpler if we statically initialize.

Also, doesn't the common case in `_PyMem_init_obmalloc()` get faster since we can skip the initialize step (e.g. the `memcpy()`)?